content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -144,6 +145,56 @@ client.query("SELECT 1;")
 
 {{% /tab %}}
 
+{{% tab "NodeJS" %}}
+
+Install or udpate [dd-trace-js][1] to version greater than `3.9.0` (or `2.22.0` if using end-of-life Node.js version 12):
+
+```
+npm install dd-trace@3.9.0
```

Comment:
I'd recommend updating this to say:
```
npm install dd-trace@^3.9.0
```
That way the user will get 3.9.1 or 3.10.0 when they come out.